resources/lang/en-US/general.php

```diff
@@ -156,10 +156,9 @@
     'image_delete'         	=> 'Delete Image',
     'include_deleted'       => 'Include Deleted Assets',
     'image_upload'         	=> 'Upload Image',
-    'filetypes_accepted_help'    => 'Accepted filetype is :types. Max upload size allowed is :size.|Accepted filetypes are :types. Max upload size allowed is :size.',
     'filetypes_size_help'   => 'Max upload size allowed is :size.',
-    'image_filetypes_help'  => 'Accepted filetypes are jpg, webp, png, gif, svg, and avif. Max upload size allowed is :size.',
-    'unaccepted_image_type'  => 'This image file was not readable. Accepted filetypes are jpg, webp, png, gif, and svg. The mimetype of this file is: :mimetype.',
+    'filetypes_accepted_help'        => 'Accepted filetypes are :filetypes. Max upload size allowed is :size.',
```

Comment:
I think you dropped off the pluralization that we previously had with the previous version of `filetypes_accepted_help'`.

Reply:
I changed it to read "Accepted filetype/s..." since backups accepts a singular type and other places accept multiple types.

Reply:
Nope :P We need to use the pluralized version we had before, divided by a pipe.

Reply:
https://github.com/snipe/snipe-it/blob/7464c3622e573420e19b964239357e1a40b4af11/resources/lang/en-US/general.php#L159 <-- that should already work

Reply:
OOOH TIL.

Reply:
fixed 👍

Reply:
I think you'll still need to pass a count to those translations to make pluralization work tho. 

```php
<p class="help-block" id="uploadFile-status">{{ trans('general.filetypes_accepted_help', 1, ['filetypes' => 'jpg, webp, png, gif, svg, and avif', 'size' => Helper::file_upload_max_size_readable()]) }} {{ $help_text ?? '' }}</p>
```

since that form only allows one image upload, versus: 

```php
<p class="help-block" id="uploadFile-status">{{ trans('general.filetypes_accepted_help', 2, ['filetypes' => 'jpg, webp, png, gif, svg, and avif', 'size' => Helper::file_upload_max_size_readable()]) }} {{ $help_text ?? '' }}</p>
```

for multiple.

It would also be great if we could pull those values right from the `rules()` in the file/image upload requests, so we don't have multiple places to keep them updated, but that's less of a hill for me to die on.

Reply:
(In this case, `1` or `2` is all you need, you don't actually need to pass it a real count, since it just needs to decide whether it should be singular or plural and the actual number doesn't matter, since we're not displaying it anywhere, versus something like "You have successfully updated 6 models", where the number `6` matters in the string.)

Reply:
We have a little more docs on pluralization here: https://snipe-it.readme.io/docs/translations

Reply:
(Heh, you thought this was going to be an easy win. Joke's on you! :P )

Reply:
added trans_choice() and the count 👍  :party-parrot: